flying-saucer-pdf/src/main/java/org/xhtmlrenderer/pdf/ITextRenderer.java

```diff
@@ -95,6 +98,13 @@ public class ITextRenderer {
     @Nullable
     private Character _pdfVersion;
 
+    @Nullable
+    private PdfPageEvent pdfPageEvent;
+    @NonNull
+    private Dimension _dim;
```

Comment:
Actually, `_dim` is nullable. Until method `layout()` is called, its value is `null`.